src/Startup.cs

```diff
@@ -46,16 +47,18 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory)
         {
+            loggerFactory.AddFile("Logs/ePiggy-{Date}.txt");
+            
             if (env.IsDevelopment())
             {
-                app.UseDeveloperExceptionPage();
+                app.UseExceptionHandler("/ErrorPages/Error");
+                //app.UseDeveloperExceptionPage();
             }
             else
             {
-                app.UseExceptionHandler("/Error");
-                // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+                app.UseExceptionHandler("/ErrorPages/Error");
                 app.UseHsts();
```

Comment:
Could you explain what does this do? I understand that it sets different settings if it's development environment or not, but the details I don't know about

Reply:
Basically it sets that when uncatched exception happens it redirects to certain page. DeveloperExceptionPage is one I think you're familiar with (where all stack traces printed, one that was shown during all exceptions). And since we're testing inside Visual studio it uses developer enviroment, so I had to comment out, and what is inside else is what would be displayed in production version (same thing).

_Hsts is a security feature to force SSL. It has nothing to do with exceptions._ - [Source](https://stackoverflow.com/questions/51093139/what-is-the-difference-between-app-usehsts-and-app-useexceptionhandler)